spec/system/partners/profile_radio_spec.rb

```diff
@@ -0,0 +1,92 @@
+RSpec.describe "Partners profile radio buttons behaviour", type: :system, js: true do
```

Comment:
This can be a request spec. System specs use a full browser and are only needed when testing interactions.

Reply:
I removed the system spec and re-implemented some of these tests as request spec.